contrib/node/src/python/pants/contrib/node/tasks/node_resolve_local.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import os
+
+from pants.base.workunit import WorkUnitLabel
+
+from pants.contrib.node.tasks.node_paths_local import NodePathsLocal
+from pants.contrib.node.tasks.node_resolve import NodeResolve
+
+
+class NodeResolveLocal(NodeResolve):
```

Comment:
Ditto subclassing concrete classes: rather than subclassing a concrete class, you should almost always prefer to extract an abstract base class with two (in this case) subclasses. That helps you to define the API, and clarifies why the classes actually need to be different (basically: it's exactly the abstract members).